src/Keboola/DbExtractor/Extractor/Snowflake.php

```diff
@@ -226,26 +241,33 @@ private function exportAndDownload(array $table): int
 
         file_put_contents(
             $outputDataDir . '.manifest',
-            Yaml::dump($this->createTableManifest($table, array_map(
```

Comment:
functional change inside CI&CS PR :(

Reply:
Should have been part of the PR that introduced a change that requires this or it's own separate PR.

Reply:
yes, it should be part of https://github.com/keboola/db-extractor-snowflake/pull/67